pytorch_lightning/metrics/functional/classification.py

```diff
@@ -252,6 +257,12 @@ def precision_recall(
         (tensor(0.5000), tensor(0.3333))
 
     """
+    rank_zero_warn(
```

Comment:
I actually see a value in having a combined function for calculating precision and recall. If I want both, then this function is approx twice as fast as calculating them individually as it is calculating the stat-scores that is the expensive part.

Reply:
Well, that's true. But this argument can be made even more widely, as not only precision and recall, but also FBeta and IoU (conceptually, and more concretely in future PRs) are based on stat scores. So maybe we should have something like [this](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_fscore_support.html)?

Anyways, if you want for now I can keep this one (but update it)

Reply:
Again, good point. I would probably be a good idea to have something like `precision_recall_fscore_support` in the future. But lets then keep the `precision_recall` functional metric, and then it can be extended to (in a future PR) to `precision_recall_fscore_support`.

Reply:
Okay, I added `precision_recall`